app/models/customer.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: customers
+#
+#  id              :integer          not null, primary key
+#  name            :string(255)      not null
+#  organization_id :integer
+#  created_at      :datetime
+#  updated_at      :datetime
+#
+
+class Customer < ActiveRecord::Base
+  belongs_to :organization, inverse_of: :customers
+  has_many :transactions, dependent: :destroy, inverse_of: :customer
+
+  validates :name, presence: true
+  validates :organization, presence: true
+  validates :organization_id, uniqueness: { scope: :name }
```

Comment:
@Gataniel  @shir I think it should be `validates :name, uniqueness: { scope: :organization_id }`, am I right?